py-polars/tests/unit/operations/unique/test_unique_counts.py

```diff
@@ -43,3 +43,27 @@ def test_unique_counts_null() -> None:
     s = pl.Series([None, None, None])
     expected = pl.Series([3], dtype=pl.UInt32)
     assert_series_equal(s.unique_counts(), expected)
+
+
+def test_unique_counts_on_bool() -> None:
+    s = pl.Series("bool", [True, False, False, True, True, True])
+    expected = pl.Series("bool", [4, 2], dtype=pl.UInt32)
+    assert_series_equal(s.unique_counts(), expected)
+
+
+def test_unique_counts_on_bool_only_true() -> None:
+    s = pl.Series("bool", [True, True, True])
+    expected = pl.Series("bool", [3], dtype=pl.UInt32)
+    assert_series_equal(s.unique_counts(), expected)
+
+
+def test_unique_counts_on_bool_only_false() -> None:
+    s = pl.Series("bool", [False, False, False])
+    expected = pl.Series("bool", [3], dtype=pl.UInt32)
+    assert_series_equal(s.unique_counts(), expected)
+
+
+def test_unique_counts_on_bool_include_null() -> None:
+    s = pl.Series("bool", [True, None, False, False, True, True])
+    expected = pl.Series("bool", [3, 1, 2], dtype=pl.UInt32)
+    assert_series_equal(s.unique_counts(), expected)
```

Comment:
Can you add a test with all nulls?

Reply:
Can you parametrize the tests? It's a lot more compact, something like:

```python
@pytest.mark.parametrize(
    ("input", "expected"),
    [
        ([True, True, True], [3]),
        ([False, False, False], [3]),
        ([False, False, False], [3]),
        ([True, False, False, True, True], [3, 2]),
    ]
)
fn test_unique_counts_bool(input: list[bool], expected: list[int]):
    assert_series_equal(
        pl.Series(input).unique_counts(),
        pl.Series(expected, dtype=pl.Int32)
    
```

Reply:
That's much cleaner indeed, made the change as requested